src/webview/columns/data-column.tsx

```diff
@@ -82,13 +84,14 @@ export class EditableDataColumnRow extends React.Component<EditableDataColumnRow
     }
 
     protected renderGroup(maus: React.ReactNode, startAddress: bigint, endAddress: bigint): React.ReactNode {
+        const breakpointMetadata = breakpointService.metadata(toHexStringWithRadixMarker(startAddress));
         return <span
-            className='byte-group hoverable'
+            className={classNames('byte-group', 'hoverable', ...BreakpointService.inlineClasses(breakpointMetadata))}
             data-column='data'
             data-range={`${startAddress}-${endAddress}`}
             key={startAddress.toString(16)}
             onDoubleClick={this.setGroupEdit}
-            {...createGroupVscodeContext(startAddress, toOffset(startAddress, endAddress, this.props.options.bytesPerMau * 8))}
+            {...createGroupVscodeContext(startAddress, toOffset(startAddress, endAddress, this.props.options.bytesPerMau * 8), breakpointMetadata)}
```

Comment:
This also suggests that context menu context generation should perhaps be a contributable activity rather than hardcoded in a single function.